magento/adobe-stock-integration
# Show alternative license popup view on zero credit

### Description (*)
This PR introduces an alternative popup view for licensing image in case there are no credits left. In this case, user will get a popup that recommends buying more credits and points the user to the corresponding URL

### Fixed Issues (if relevant)
1. #485 : Licensing image. Out of credits message.

### Manual testing scenarios (*)
1. Go to Admin -> Content -> Pages -> Choose a page | create new
2. Open dialog for inserting an image to a content
3. Click on "Open Adobe Stock"
4. Choose an image
5. Click on "License and Save" button
6. Log into the Adobe Stock account that has no credits
7. You should see a popup that recommends buying more credits

![image](https://user-images.githubusercontent.com/2148959/65763722-af74f880-e12c-11e9-8b41-eff56e07e603.png)

8. By clicking on the "Buy Credits" a new tab is being appeared that goes to https://stock.adobe.com/
